### PR TITLE
Arrow spacing rule

### PR DESCRIPTION
**Pull request by martin91s**
## Suggested rule/changes(s):
* `<arrow-spacing>` 

## Reason for addition/amendment

Before and after the arrow function's arrow, should always include spacing. 
for example: (a) => {}.

Both before and after should be set as true.
{ "before": true, "after": true }.

This helps the code to be more readable.

Feel free to include code samples if it's a new rule.

@netsells/frontend - Please review

### CODE REVIEW

File: rules/javascript.js [hunk 1]
```diff
@@ -21,5 +21,7 @@ module.exports = {
             matchDescription: ".+",
             requireReturn: false,
         }],
+        //
```
Comment (samturrell) [inline rules/javascript.js]: Give a quick blurb :D

File: rules/javascript.js [hunk 2]
```diff
@@ -21,5 +21,7 @@ module.exports = {
             matchDescription: ".+",
             requireReturn: false,
         }],
+        // 
+        'arrow-spacing': [_THROW.ERROR, {before: true, after: true}],
```
Comment (samturrell) [inline rules/javascript.js]: Drop things down onto new lines e.g.:
```js
'arrow-spacing': [_THROW.ERROR, {
    before: true, 
    after: true,
}],
```

**Review by samturrell — CHANGES_REQUESTED** [review]

File: readme.md [hunk 3]
```diff
@@ -165,18 +163,40 @@ methods: {
 }
 ```
 
+<<<<<<< HEAD
```
Comment (samturrell) [inline readme.md]: Nooooooooooooooooo

File: readme.md [hunk 4]
```diff
@@ -165,18 +163,40 @@ methods: {
...
+Requires space before and after arrow functions
+=======
 #### 📍 no-var
 Discourages using `var` for creating variables and requires using `let` or `const` instead
+>>>>>>> master
 
 ##### ❌ Example of incorrect code for this rule:
 
 ```
+<<<<<<< HEAD
```
Comment (samturrell) [inline readme.md]: noooooooooo

File: readme.md [hunk 5]
```diff
@@ -165,18 +163,40 @@ methods: {
...
+
+=======
 var count = posts.length;
+>>>>>>> master
 ```
 
 ##### ✅ Example of correct code for this rule:
 
 ```
+<<<<<<< HEAD
```
Comment (samturrell) [inline readme.md]: Nooooooooo

**Review by samturrell — APPROVED** [review]

File: readme.md (hunk 3 above)
Comment (wik-z) [reply readme.md]: Nooooooooooooooooo

File: readme.md [hunk 6]
```diff
@@ -169,16 +167,10 @@ methods: {
 Discourages using `var` for creating variables and requires using `let` or `const` instead
 
 ##### ❌ Example of incorrect code for this rule:
-
-```
-var count = posts.length;
-```
+`var count = posts.length;
```
Comment (samturrell) [inline readme.md]: I don't think you've fixed this properly

**Review by samturrell — CHANGES_REQUESTED** [review]

**Review by samturrell — APPROVED** [review]

**Pull request merged**